### PR TITLE
bitrise 1.3.7

### PR DESCRIPTION
**Pull request by godrei**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

File: Formula/bitrise.rb [hunk 1]
```diff
@@ -12,17 +12,6 @@ class Bitrise < Formula
   end
 
   depends_on "go" => :build
-  depends_on "godep" => :build
```
Comment (dunn) [inline Formula/bitrise.rb]: How are dependencies managed now?

**Comment by DomT4** [comment]
May well need a `prefix.install_metafiles` inside the `cd bitrise_go_path do` block here.

File: Formula/bitrise.rb (hunk 1 above)
Comment (viktorbenei) [inline Formula/bitrise.rb]: `vendor`d in the repo https://github.com/bitrise-io/bitrise

`stepman` and `envman` are downloaded on the fly, when required, into `$HOME/.bitrise/tools`

**Comment by viktorbenei** [comment]
@DomT4 why? What does that do?

**Comment by DomT4** [comment]
I guess the confused response is what I deserve for referring casually to a piece of Homebrew's internal mechanism. Homebrew has "magic" around to ensure that we install things like `LICENSE`, `README`, `COPYING` etc into the `prefix` from the `buildpath`.

Sometimes when the build happens in somewhere that isn't the top-level `buildpath` that logic doesn't kick in, and `prefix.install_metafiles` is just a mechanism to say _"Hey, don't forget to check the current directory for those things"_.

**Comment by viktorbenei** [comment]
@DomT4 ohh, makes sense, I wasn't aware of `install_metafiles`, or what "meta" meant here - we'll of course add it to the `bitrise_go_path` block asap, thanks for the explanation!

**Comment by godrei** [comment]
Hi @DomT4,
i added the `prefix.install_metafiles` in the `cd bitrise_go_path do` block, but i just noticed bitrise 1.3.7 is now available in brew, but this PR was not closed.
Should i create a new version with these changes?

**Comment by DomT4** [comment]
Oh, yeah, looks like it was merged in https://github.com/Homebrew/homebrew-core/commit/1244b2b02dc42065c766091bd31029492553b2d9 but our usual `Closes #xyz` tag that ensures completed PRs aren't left open wandered off for some reason. Sorry about that.

If you wouldn't mind submitting a new PR with the `prefix.install_metafiles` tweak that'd be great, Thanks!

**Comment by godrei** [comment]
@DomT4 
the new PR: https://github.com/Homebrew/homebrew-core/pull/3768

**Pull request closed without merging**